ofl/notosanssignwriting/METADATA.pb

```diff
@@ -0,0 +1,22 @@
+name: "Noto Sans SignWriting"
+designer: "Google"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2022-09-12"
+fonts {
+  name: "Noto Sans SignWriting"
+  style: "normal"
+  weight: 400
+  filename: "NotoSansSignWriting-Regular.ttf"
+  post_script_name: "NotoSansSignWriting-Regular"
+  full_name: "Noto Sans SignWriting Regular"
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/sign-writing)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+source {
+  repository_url: "https://github.com/notofonts/sign-writing.git"
+  archive_url: "https://github.com/notofonts/sign-writing/releases/download/NotoSansSignWriting-v2.004/NotoSansSignWriting-v2.004.zip"
+}
+is_noto: true
```

Comment:
Should there be subset, language, sample text defined for this?

Reply:
I'll add some sample text.

Reply:
Actually it looks like the font is not working properly anyway.